cmake/modules/Fortran/Cray.cmake

```diff
@@ -9,7 +9,7 @@
 
 # General Flags (add to default)
 set(CMAKE_Fortran_FLAGS 
-    "${CMAKE_Fortran_FLAGS} -ffree -eZ -N255 -ec -eC -eI -eF -hnosecond_underscore -hflex_mp=conservative -Ofp1 -hadd_paren -ra")
+    "${CMAKE_Fortran_FLAGS} -ffree -N255 -ec -eC -eI -eF -hnosecond_underscore -hflex_mp=conservative -Ofp1 -hadd_paren -ra")
```

Comment:
Removing "`-eZ`" which forces preprocessing, but there's nothing to preprocess in any of the files. By default if you want preprocessing, change the file extension from `.f90` to `.F90`  and the compiler will do the right thing.
The compiler seemed to fall over until this flag was removed.